src/style.css

```diff
@@ -1 +1,132 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
```

Comment:
The font-weight for 'Avenir-Book' is set to 300, which is typically considered 'light'. Ensure this matches the actual weight of the font file to avoid rendering issues.